changelog.md

```diff
@@ -1,4 +1,13 @@
 # Changelog
+
+## v.3.4.0
+Added the option to reset user optout status on newsletter subscription. Appears as a checkbox in the footer widget and as the following option in the shortcode:
+```
+[sailthru_widget ... reset_optout_status="true"]
+```
+
+Enabling this option will change the user's optout status to "valid" in Sailthru by passing the `optout_email=none` option in the API.
```

Comment:
@brownac Just double checking here, (based on the version increment) that this change adds functionality in a backwards compatible manner.

https://semver.org/
```
Given a version number MAJOR.MINOR.PATCH, increment the:

MAJOR version when you make incompatible API changes,
MINOR version when you add functionality in a backwards compatible manner, and
PATCH version when you make backwards compatible bug fixes.
```